zod/constants.py

```diff
@@ -97,3 +97,7 @@ class Lidar(Enum):
 
 
 CoordinateFrame = Union[Camera, Lidar, Literal[EGO]]
+
+# ZodFrame properties
+MEAN = [0.337, 0.345, 0.367]
```

Comment:
Can you change the variable names to something more specific; If we ever want to add the same for ZodSequences we have to change the names.

Suggestion: 
FRAMES_IMAGE_MEAN
FRAMES_IMAGE_STD

Reply:
Of course! Force pushed the proposed changes 🚀